warehouse/query-core/src/main/java/datawave/query/jexl/visitors/LiteralNodeVisitor.java

```diff
@@ -35,4 +38,23 @@ public Object visit(ASTEQNode node, Object data) {
         return literals;
     }
     
+    // Descend through these nodes
+    @Override
+    public Object visit(ASTJexlScript node, Object data) {
+        node.childrenAccept(this, data);
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTERNode node, Object data) {
```

Comment:
My bad, this one needs the ASTNRNode as well I think.

Reply:
Updated